test/orchard/info_test.clj

```diff
@@ -368,6 +366,25 @@
                     (map #(info/info* %))
                     (map #(select-keys % [:ns :name :arglists :macro :file])))))))))
 
+(deftest info-no-file-info-test
+  (testing "File info key does not exist should not resolve classpath - issue #61"
+    (let [params '{:sym finally}
+          expected '{:forms [(try expr* catch-clause* finally-clause?)],
+                     :doc "catch-clause => (catch classname name expr*)\n  finally-clause => (finally expr*)\n\n  Catches and handles Java exceptions.",
+                     :name finally,
+                     :special-form true,
+                     :url "https://clojure.org/special_forms#finally"}]
+      (testing "- boot project"
+        (with-redefs [orchard.misc/boot-project? (constantly true)]
+          (let [i (info/info* params)]
+            (is (= expected (select-keys i [:ns :name :doc :forms :special-form :url])))
+            (is (nil? (:file i))))))
+
+      (testing "- no boot project"
```

Comment:
Is it just me or are there no differences in the two tests expect the type of the project?

Reply:
uhm, so I am testing that the `:file` key is not present AND that of course no exception is thrown.

There is a `with-redefs` for `boot` projects and I tested with and without the patch (written the tests first).

Do you have any suggestion for improving it or maybe I am missing something?

Reply:
I just assumed that because you factored in the project type you were expecting it to be meaningful to those tests. That's why I asked.

Reply:
Yes the test is making sure we don't have exceptions in both with and without boot

Reply:
OK, fair enough.